subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -157,6 +162,31 @@ int nrf_cloud_obj_msg_init(struct nrf_cloud_obj *const obj, const char *const ap
 		obj->json = new_obj;
 		return 0;
 	}
+	case NRF_CLOUD_OBJ_TYPE_COAP_CBOR:
+	{
+		if (obj->coap_cbor) {
+			return -ENOTEMPTY;
+		}
+
+		size_t app_id_sz = strlen(app_id) + 1;
+		struct nrf_cloud_obj_coap_cbor *msg = nrf_cloud_calloc(1, sizeof(*msg));
+
+		if (!msg) {
+			return -ENOMEM;
+		}
+
+		/* Add the app_id */
```

Comment:
initialize`msg->type` 
e.g.: `msg->type = NRF_CLOUD_DATA_TYPE_NONE;`

Reply:
It's redundant since calloc() sets it to 0, and NRF_CLOUD_DATA_TYPE_NONE = 0, but if you want it for clarity I can.

Reply:
yes, it would be good to set it explicitly.
i don't know what the value of `NRF_CLOUD_DATA_TYPE_NONE` since it doesn't exist yet... and maybe its value will be `-1` or `0xFFFFFFFF`.